apps/app/locals.tf

```diff
@@ -14,4 +14,5 @@
 
 locals {
   feature_store_odbc = "Driver={ODBC Driver 18 for SQL Server};Server=tcp:${data.azurerm_mssql_server.feature_store.fully_qualified_domain_name},1433;Database=${var.feature_store_db_name};Authentication=ActiveDirectoryMsi;Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;"
+  is_prod            = length(regexall("prod", lower(var.environment))) > 0
```

Comment:
Assuming this is temporary? This will be populated by the app config instead of by the environment itself - I was thinking of a slots property as per https://github.com/UCLH-Foundry/Book-of-FlowEHR/blob/the-walking-skeleton/milestones/walking-skeleton/building-clinical-product/serve-devops-design.md#3-staging-within-the-production-environment---production